### ISSUE TITLE
Personnalisation des fonds de cartes

### ISSUE THREAD

**Issue by xavyeah39** [Georiviere/Georiviere-public#52]
Bonjour,

Je retrouve pas (ni dans ma mémoire ni dans la doc) le paramètrage permettant de personnaliser les layers des fonds de cartes.
Pouvez-vous m'éclairer ? 
Merci !

**Comment by babastienne** [comment]
Bonjour, la paramétrage des fonds de carte est identique à Geotrek, c'est pour ça que la doc de GeoRiviere fait simplement référence à la doc de Geotrek sur ce point, sûrement un aspect qui pourrait être éclairci. 

Toutes les infos pour modifier les layers sont disponibles ici : https://geotrek.readthedocs.io/en/2.102.1/install/advanced-configuration.html#map-settings

**Comment by xavyeah39** [comment]
Salut Bastien, OK mais ça pour l'admin pas pour Georiviere-public ?

**Comment by babastienne** [comment]
Ah oui, au temps pour moi, j'ai confondu les briques ... :laughing: 

Les fonds de plan de GeoRiviere Public peuvent être gérés directement dans l'interface d'administration de l'admin, via le lien suivant : https://monserveur.com/admin/portal/mapbaselayer/

**Comment by xavyeah39** [comment]
Top merci ! 👍
Comme le fonctionnement diffère un peu de Geotrek ce serait utile qu'on ajoute un bout de doc côté GR-public non ?

**Comment by babastienne** [comment]
Oui complètement d'accord, il faut le documenter !

**Comment by xavyeah39** [comment]
Yes ! Je veux bien m'y coller quand j'ai un moment mais je note quand même un autre soucis que je met dans une issue distincte : #53
